### PR TITLE
CSP: Print fixed traceback in return error

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
instead of printing the traceback in each integration/script (what we have now - see screenshots), the `return_error` will do it.

## Screenshots
will have bad traceback line numbers
image
will have fixed traceback line numbers
image

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
The reason we don't do it is cause to looks very very ugly. I want it only in the logs, not in the war-room.

**Comment by yaakovi** [comment]
> The reason we don't do it is cause to looks very very ugly. I want it only in the logs, not in the war-room.

it will still be only in the logs, I didn't add it to the entry

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -6395,7 +6397,7 @@ def return_error(message, error='', outputs=None):
             'Type': entryTypes['error'],
             'ContentsFormat': formats['text'],
             'Contents': message,
-            'EntryContext': outputs
+            'EntryContext': outputs,
```
Comment (ShahafBenYakir) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:6400]: 🤔 ?

**Review by ShahafBenYakir — APPROVED** [review]
👍

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py (hunk 1 above)
Comment (yaakovi) [reply Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:6400]: [PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)

**Pull request merged**